lib/stub_ui/lib/src/ui/text.dart

```diff
@@ -767,13 +766,15 @@ class StrutStyle {
   /// * `fontSize`: The size of glyphs (in logical pixels) to use when painting
   ///   the text.
   ///
-  /// * `lineHeight`: The minimum height of the line boxes, as a multiple of the
-  ///   font size. The lines of the paragraph will be at least
-  ///   `(lineHeight + leading) * fontSize` tall when fontSize
-  ///   is not null. When fontSize is null, there is no minimum line height. Tall
-  ///   glyphs due to baseline alignment or large [TextStyle.fontSize] may cause
-  ///   the actual line height after layout to be taller than specified here.
-  ///   [fontSize] must be provided for this property to take effect.
+  /// * `height`: The minimum height of the line boxes, as a multiplier of the
+  ///   font size. The lines of the paragraph will be at least `(height + leading)
+  ///   * fontSize` tall when fontSize is not null. Omitting `height` will allow
+  ///   the minimum line height to take the height as defined by the font, which
+  ///   may not be exactly the height of the fontSize. When fontSize is null, there
```

Comment:
should the fontSizes mentioned in this paragraph be enclosed in ` or []?

Reply:
They are both parameters as well as fields...

I'll use [] because it'll provide a link with more info, whereas `` will not.

Reply:
[] is the right choice if they are fields.

Reply:
Jk I lied, they are not fields at this level of the API. I'll use ``